cmd/gitcloner/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"os"
+
+	"github.com/rancher/gitjob/cmd/gitcloner/cmd"
+	"github.com/rancher/gitjob/cmd/gitcloner/gogit"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	logrus.Info("Starting to clone git repository")
+	cmd := cmd.New(gogit.NewCloner())
+	err := cmd.Execute()
+
+	if err != nil {
```

Comment:
Nit-pick: [Effective go style guides](https://go.dev/doc/effective_go) prefers a compact error check:
```suggestion
	if err := cmd.Execute(); err != nil {
```